### ISSUE TITLE
feature: backlinks

### ISSUE THREAD

**Issue by majidaldo** [twinkle-labs/twinkle-notes#7]
When a note is serving as a 'tag' (or some kind of stub), it's useful to see other notes that are referring to it.

Should be easy since I'm assuming is just a matter of searching. This is a major feature of Roam Research.

**Comment by litchie** [comment]
Yes, it should be useful to show which notes are referring to current note. It's easy to do, just searching the note id.

Tags can be specified with '#tagname'. And you can search notes that includes a certain tag.

**Comment by litchie** [comment]
Implementation note: show referencing notes along side the sub branches, perhaps in a separate tab.

**Comment by majidaldo** [comment]
just a comment: with tiddlywiki there is no distinction b/w a 'tag' and a note. they are both 'notes' that can be referenced. personally, i don't care too much about some markup to mean a 'tag'. i'd rather have 'tags' as simply another structure that i can define through notes.
